### PR TITLE
Added two new cli commands

### PR DESCRIPTION
**Pull request by marukuru**
Added two little features I found quite handy:
- Get actor's current state with switch-state <ain>
- Toggle actor's state with switch-toggle <ain>

Fixed parsing in Actor.get_state()

First time using Python, please be gentle ;-)

### CODE REVIEW

File: fritzhome/__main__.py [hunk 1]
```diff
@@ -165,6 +165,37 @@ def switch_off(context, ain):
         click.echo("Actor not found: {}".format(ain))
 
 
+@cli.command(name="switch-state")
+@click.argument('ain')
+@click.pass_context
+def switch_on(context, ain):
```
Comment (DerMitch) [inline fritzhome/__main__.py]: Please update the method name (`switch_on` -> `switch_state`)

File: fritzhome/__main__.py [hunk 2]
```diff
@@ -165,6 +165,37 @@ def switch_off(context, ain):
...
+    if actor:
+        click.echo("State for {} is: {}".format(ain,'ON' if actor.get_state() else 'OFF'))
+    else:
+        click.echo("Actor not found: {}".format(ain))
+
+
+@cli.command(name="switch-toggle")
+@click.argument('ain')
+@click.pass_context
+def switch_on(context, ain):
```
Comment (DerMitch) [inline fritzhome/__main__.py]: Please update the method name (`switch_on` -> `switch_toggle`)

**Comment by DerMitch** [comment]
Thank your for your contribution.

Please update your code according to my comments (overriding methods is not a good idea), then I'll merge it. :-)

**Comment by marukuru** [comment]
Thanks for noticing. Must have screwed up the method names while merging changes from the virtual env to the GitHub repo.

**Pull request merged**